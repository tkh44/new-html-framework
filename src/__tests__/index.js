const pluginTester = require('babel-plugin-tester')
const plugin = require('../index')

pluginTester({
  plugin: plugin,
  pluginName: 'sitrep',
  snapshot: true,
  babelOptions: {
    babelrc: true,
    filename: __filename
  },
  tests: [
    {
      title: 'basic',
      code: `
        html\`
          <div>
            <h1>Hello</h1>
          </div>
        \`;
      `
    },
    {
      title: 'with expression',
      code: `
        html\`
          <div>
            <h1>\${foo}</h1>
          </div>
        \`;
      `
    }
  ]
})
