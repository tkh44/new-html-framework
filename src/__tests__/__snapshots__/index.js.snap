// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic 1`] = `
"
html\`
  <div>
    <h1>Hello</h1>
  </div>
\`;

      ↓ ↓ ↓ ↓ ↓ ↓

<div>
    <h1>Hello</h1>
  </div>;
"
`;

exports[`with expression 1`] = `
"
html\`
  <div>
    <h1>\${foo}</h1>
  </div>
\`;

      ↓ ↓ ↓ ↓ ↓ ↓

<div>
    <h1>{foo}</h1>
  </div>;
"
`;
